kahuna/public/js/search/results.html

```diff
@@ -44,7 +44,7 @@
                                        gr:datalist-input />
                             </gr-datalist>
                         </form>
-                        <span ng:switch-default>{{ctrl.parentLabel}}:</span>
+                        <span ng:switch-default>{{ctrl.parentLabel}}</span>:
```

Comment:
I personally wouldn't expect a `:` after the input field followed by nothing, but YMMV. Up to you.